src/components/schema-table.js

```diff
@@ -25,6 +25,19 @@ export default class SchemaTable extends LitElement {
     if (!this.schemaHideWriteOnly || !'true false'.includes(this.schemaHideWriteOnly)) { this.schemaHideWriteOnly = 'true'; }
   }
 
+  firstUpdated() {
+    const observer = new IntersectionObserver((entries) => {
```

Comment:
It would be great if we added a comment here to explain why we are doing this (same goes for the other location). It would also be good to explain why we need a `300` timeout. What happens at 200ms, 100ms, 1ms, next tick of the event loop, that would be too soon. I think that's the part that feels the most dirty to me.

Reply:
Yes, I agree, it's not pretty.

The tricky part here is that it's implemented today as css animation and not transition. They play every times the dom is updated from the start of the animation definition. I'll do my best to explain it in the next commit ^^

Reply:
What if we just put the class adding into the expand/collapse button clicks. That would be better if it works.

Reply:
I have a new working solution without setTimeout and IntersectionObserver, I will propose it to you this afternoom.